src/components/Post/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.scss';
+
+const Post = ({
+  title,
+  body,
+  publishedAt,
+  author: { name },
+}) => (
+  <div className="post">
+    <div className="post__header">
+      <div className="post__header__user">
+        <div className="post__header__username">
```

Comment:
Here you are using BEEM,not BEM. BEM means Block - Element - Identifier. You have 2 E here, header, user and header, username. I know this seems like small now but it you have more nestings, it becomes bigger and bigger (i.e post_header__user__username__imagine-more__or-more).

In this case, it is better post__user (or post__user-container) and post__username.

This comment should be addressed on the whole PR

Reply:
Luciano told me that BEM is Block-Element-Modifier but he's scared to tell you that.

Reply:
Don't burn me